test/test_helper.rb

```diff
@@ -183,7 +183,11 @@
 module ActiveSupport
   class TestCase
     include ActiveRecord::TestFixtures
-    self.fixture_path = File.dirname(__FILE__) + "/fixtures"
+    if respond_to?(:fixture_paths=)
+      self.fixture_paths = [File.dirname(__FILE__) + "/fixtures"]
+    else
+      self.fixture_path = File.dirname(__FILE__) + "/fixtures"
```

Comment:
Should we explain why this is needed? Not necessarily linking https://github.com/rails/rails/pull/47675, but something like:
```rb
# fixture_path is deprecated in Rails 7.1 and removed from Rails 7.2.
# (use fixture_paths instead)
```

This is consistent with the comment here for `if respond_to?(:use_transactional_tests=)`.